plugins/modules/lke_cluster.py

```diff
@@ -133,6 +133,12 @@
         ],
         suboptions=linode_lke_cluster_autoscaler,
     ),
+    "disk_encryption": SpecField(
+        type=FieldType.string,
+        description="The disk encryption status of this node pool. "
+        + "NOTE: Disk encryption may not currently be available to all users.",
+        choices=["enabled", "disabled", "pde"],
+    ),
```

Comment:
`disk_encryption` seems a read-only field for lke_cluster. We don't have to put it as a SpecField here.

Reply:
Interesting. I updated some tests to create lke_clusters with disk_encryption set and they seemed to be working. Where is it listed that `disk_encryption` is read-only for `lke_cluster` and `lke_node_pool`?